model/registry/process.yaml

```diff
@@ -147,6 +157,15 @@ groups:
           An additional description about the runtime of the process, for example
           a specific vendor customization of the runtime environment.
         examples: 'Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0'
+      - id: title
+        type: string
+        stability: experimental
+        brief: >
+          Process title
+        note: >
+          The proctitle, some times the same as process name. Can also be different, for
```

Comment:
should it be `process title`?

Reply:
proctitle is the common term for it, for example see https://manpages.ubuntu.com/manpages/trusty/man3/setproctitle.3.html

I've reworked the note to have both terms though